CONTRIBUTING.md

```diff
@@ -1,32 +1,135 @@
+
+You'll want to install various bits of software.
 On a Debian-based OS, run:
 
 ```console
 $ sudo apt install \
     cargo \
+    gcc \
     git \
-    libapt-pkg-dev \
+    libpython3-dev \
+    libssl-dev \
+    pkg-config \
     protobuf-compiler \
-    python3-pip \
-    python3-venv \
-    rustc \
-    swig
+    python3-gpg
 ```
 
-For example, to create a development environment:
+<!--
+In order (Package, command, error):
+  - git                  $ git clone https://github.com/jelmer/janitor.git     # -bash: git: command not found
+  - protobuf-compiler    $ pip3 install --editable .                           # Getting requirements to build editable ... error -> error: Unable to find protobuf compiler protoc
+  - pkg-config           $ pip3 install --editable .                           # Collecting gpg -> /root/janitor/.venv/bin/gpgme-config: line 29: exec: pkg-config: not found
+  - libgpgme-dev         $ pip3 install --editable .                           # Collecting gpg -> Package gpgme was not found in the pkg-config search path.
+  - cargo                $ pip3 install --editable .                           # Building wheels for collected packages: janitor, aiohttp-apispec, breezy -> Building editable for janitor (pyproject.toml) ... error -> distutils.errors.DistutilsPlatformError: can't find Rust compiler
+  - gcc                  $ pip3 install --editable .                           # Building wheel for breezy (pyproject.toml) ... error -> error: command 'x86_64-linux-gnu-gcc' failed: No such file or directory
+  - libpython3-dev       $ pip3 install --editable .                           # Building wheel for breezy (pyproject.toml) ... error -> breezy/bzr/_simple_set_pyx.c:32:10: fatal error: Python.h: No such file or directory
+  - libssl-dev           $ pip3 install --editable .                           # warning: openssl-sys@0.9.105: Could not find directory of OpenSSL installation, and this `-sys` crate cannot proceed without this knowledge. If OpenSSL is installed and this crate had trouble finding it,  you can set the `OPENSSL_DIR` environment variable for the compilation process. See stderr section below for further information.
+  - g++                  $ pip3 install --editable .                           # error: linking with `cc` failed: exit status: 1 -> distutils.errors.CompileError: `cargo build --manifest-path worker/Cargo.toml --message-format=json-render-diagnostics -v --features 'cli debian'` failed with code 101
+  - swig                 $ pip3 install --editable .                           # Running setup.py install for gpg ... error -> Using gpgme.h from /usr/include/gpgme.h -> error: command 'swig' failed: No such file or directory
+
+- Using python3-gpg over PyPi GPG as having issues with Debian Testing (okay with Debian 12)
+  https://github.com/jelmer/janitor/issues/931
+
+Dependencies
+  - cargo -> rustc
+-->
+
+- - -
+
+Even when using the latest [Debian](https://tracker.debian.org/pkg/rustc)
```

Comment:
I'd rather just focus on Debian testing. The janitor is intended to be packaged there should be compatible with the rustc in testing too.

Reply:
Dropped!